admin/app.js

```diff
@@ -218,6 +219,7 @@ app.use('/pupil-pin', pupilPin)
 app.use('/api/questions', questions)
```

Comment:
i desperately want to change this to `/api/pupil-login` but i think we should leave it until we extract out the endpoints into separate app(s)

Reply:
@GuyHarwood yea it is outside of the scope of the ticket anyway. we'll correct this on the new app.